devel/cpptoml/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        skystrife cpptoml 0.1.1 v
+checksums           rmd160  8471ec7641bccc67299cef3ddcf1b5f113fd6dbe \
+                    sha256  23af72468cfd4040984d46a0dd2a609538579c78ddc429d6b8fd7a10a6e24403 \
+                    size    47398
+
+categories          devel
+license             MIT
+
+maintainers         nomaintainer
+
+description         A header-only library for parsing TOML configuration files.
+long_description    {*}${description}
+
+github.tarball_from archive
+
+# depends_lib-append  port:bison \
+#                     port:python38 \
+#                     port:flex \
+#                     port:gflags \
+#                     port:google-glog \
+#                     port:double-conversion \
+#                     port:libevent \
+#                     port:libsodium \
+#                     port:snappy \
+#                     path:lib/libssl.dylib:openssl \
+#                     port:mstch \
+#                     port:wangle \
+#                     port:folly \
+#                     port:fizz \
+#                     port:fbthrift \
+#                     port:py38-thrift \
+#                     port:zstd \
+#                     port:zlib
+
+compiler.cxx_standard   2017
+
+# configure.args-append \
+#                     -DBUILD_TESTS=OFF \
+#                     -DBUILD_SHARED_LIBS=ON \
+#                     -DPYTHON_EXTENSIONS=OFF
```

Comment:
please don't add all these commented lines; either they are needed and should not be comments or they are not needed.